notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1551 @@
+{
```

Comment:
<p>A small improvement can be made for readability. Instead of using <code>if os.path.exists(code_dir+'hst1pass.e') is False</code>  it's more readable to use </p><p><code>if not os.path.exists(code_dir+'hst1pass.e'):</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
Personally I find the existing line most readable, as it makes it clear that `os.path.exists` returns a boolean value.

I would like to retain this line, but if you disagree for best coding practices please let me know and I'll change it?